cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,101 @@
 /// <reference types='cypress' />
 
-describe('Web Tables page', () => {
-  it('', () => {
+describe('Web Tables - Pagination Test', () => {
```

Comment:
you can use move all your test into one describe and provide the clear summary for each test

Reply:
Fixed